charts/team-settings/values.yaml

```diff
@@ -9,7 +9,7 @@ resources:
     cpu: "1"
 image:
   repository: quay.io/wire/team-settings
-  tag: "4.2.0-v0.28.28-1e2ef7"
+  tag: "4.3.0-v0.28.29-0-a2f11cf"
```

Comment:
@Yserz the automatic PR made here uses a tag that doesn't exist on quay.io:

I can see these last versions on https://quay.io/repository/wire/team-settings?tab=tags;

```
production
4.2.0-v0.28.28-f58c700
staging
4.2.0-v0.28.28-8f89a7e
...
```

Can something be done on your side with respect to align this versioning?

Reply:
Has this issue been fixed by https://github.com/wireapp/wire-team-settings/pull/5037 ? If so can you please retrigger a new PR @Yserz ?